flakeheaven/_logic/_extractors.py

```diff
@@ -401,3 +405,23 @@ def extract_wemake_python_styleguide() -> Dict[str, str]:
             code = 'WPS' + str(checker.code).zfill(3)
             codes[code] = checker.error_template
     return codes
+
+
+def extract_flake8_pie() -> Dict[str, str]:
+    import flake8_pie
+    from inspect import getsource
```

Comment:
I'd move the imports to the top of the file

Reply:
I could be wrong, but I think this was intentional so that the module is not dependent on this `flake8_pie` package unless this function is called, the same pattern seems repeated across this file

Reply:
I will leave this unchanged because that is how the other plugins were handled and @jarojasm95 is likely correct about it being intentional.